### PR TITLE
O11y - Handle edge cases for detecting package version

### PR DESCRIPTION
**Pull request by amaanbs**
Detect package version from multiple sources if running tests on Browserstack Automate.

### CODE REVIEW

**Review by sriteja777 — APPROVED** [review]

File: bin/testObservability/crashReporter/index.js [hunk 1]
```diff
@@ -47,9 +48,43 @@ exports.requireModule = (module, internal = false) => {
...
   if(packages[package_]) return packages[package_];
-  return packages[package_] = this.requireModule(`${package_}/package.json`).version;
+  let packageVersion;
+  /* Try to find version from module path */
+  try {
+    packages[package_] = this.requireModule(`${package_}/package.json`).version;
+    logger.info(`Getting ${package_} package version from module path = ${packages[package_]}`);
+    packageVersion = packages[package_];
+  } catch(e) {
+    exports.debug(`Unable to find package ${package_} at module path with error ${e}`);
```
Comment (roshan04) [inline bin/testObservability/crashReporter/index.js:60]: wont it just print [OBJECT][OBJECT] and not the error message ?

File: bin/testObservability/helper/helper.js [hunk 2]
```diff
@@ -351,9 +351,43 @@ const getCiInfo = () => {
...
   if(packages[package_]) return packages[package_];
-  return packages[package_] = this.requireModule(`${package_}/package.json`).version;
+  let packageVersion;
+  /* Try to find version from module path */
+  try {
+    packages[package_] = this.requireModule(`${package_}/package.json`).version;
+    logger.info(`Getting ${package_} package version from module path = ${packages[package_]}`);
+    packageVersion = packages[package_];
+  } catch(e) {
+    exports.debug(`Unable to find package ${package_} at module path with error ${e}`);
```
Comment (roshan04) [inline bin/testObservability/helper/helper.js:363]: wont it just print [OBJECT][OBJECT] and not the error message ?

File: bin/testObservability/crashReporter/index.js (hunk 1 above)
Comment (amaanbs) [reply bin/testObservability/crashReporter/index.js]: Nope it prints the first line of the error stacktrace.

File: bin/testObservability/helper/helper.js (hunk 2 above)
Comment (amaanbs) [reply bin/testObservability/helper/helper.js:363]: Nope it prints the first line of the error stacktrace.

**Review by roshan04 — APPROVED** [review]
